style.css

```diff
@@ -82,23 +127,263 @@
   margin-right: 30px;
 }
 
-.social-media-2 {
+.menu-list {
   display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 64px 24px;
+  gap: 58px;
+  min-width: 379px;
+  height: 3283px;
+  top: 708px;
+}
+
+.works-heading {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 30px;
+  line-height: 52px;
+  min-width: 327px;
+  height: 97px;
+  text-align: center;
+  color: #172b4d;
+}
+
+.work-profiles {
+  display: grid;
+  grid-template-rows: auto auto auto auto auto auto;
+  grid-gap: 50px;
+}
+
+.work-profile-1 {
+  display: grid;
+  min-width: 327px;
+  justify-items: center;
+  grid-template-rows: repeat(2, 1fr);
+  height: 474px;
+  background-image: url(./Images/Img-Plaholder.png);
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.works-text {
+  grid-row: 2;
+  background: #fff;
+  border-radius: 8px;
+  width: 291px;
+  height: 203px;
+  left: calc(50% - 327px / 2);
+  bottom: 0;
+  padding: 16px;
+  gap: 16px;
+}
+
+.works-title {
+  text-align: center;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  color: #3a4a42;
+  font-weight: 600;
+  font-size: 20px;
+  line-height: 32px;
+  width: 295px;
+  height: 57px;
+}
+
+.works-tags {
+  min-width: 305px;
+  height: 40px;
+}
+
+.tags {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+  gap: 8px;
+  margin-right: 40px;
   list-style-type: none;
+}
+
+.tech-tag {
+  height: 20px;
+  background: #ebf0ee;
+  border-radius: 4px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 10px;
+  line-height: 16px;
+  color: #3a4a42;
+  padding: 5px 5px;
+}
+
+.works-cta {
+  width: 305px;
+  height: 69px;
+  display: flex;
+  flex-direction: row;
   justify-content: center;
-  gap: 20px;
-  margin-right: 30px;
+  align-items: center;
+  border-radius: 4px;
+  bottom: 8px;
+  padding: 12px;
+  gap: 10px;
 }
 
-.social-media-3 { display: none; }
+.project-button {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 12px;
+  gap: 10px;
+  width: 122px;
+  height: 48px;
+  left: calc(50% - 122px / 2 + 0.5px);
+  bottom: 8px;
+  background: #36b37f;
+  border-radius: 4px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  color: #fff;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  margin-right: 45px;
+  margin-bottom: 20px;
+  border: #fff;
+}
 
-.menu-list {
+.about-page {
   display: flex;
   flex-direction: column;
   align-items: center;
   padding: 64px 24px;
   gap: 58px;
-  width: 379px;
+  min-width: 379px;
   height: 3283px;
   top: 708px;
 }
```

Comment:
Same here kindly make sure that you are using the same margins as provided by the Figma template on both sides. :eyes :eyes: 
![image](https://user-images.githubusercontent.com/98363075/193418021-8275b799-b509-4a09-b97a-003e498cdcbd.png)